test/plugins/scenarios/metric-activation/config/excepted.yml

```diff
@@ -0,0 +1,61 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+segmentItems: []
+meterItems:
+  - serviceName: metric-activation
+    meterSize: ge 4
+    meters:
+      - meterId:
+          name: request_counter
+          tags: []
+        singleValue: ge 1
+      - meterId:
+          name: request_counter_with_label
+          tags:
+            - { name: foo, value: bar }
+        singleValue: ge 1
+      - meterId:
+          name: cpu_usage_gauge
+          tags: []
+        singleValue: ge 50
+      - meterId:
+          name: cpu_usage_gauge_with_label
+          tags:
+            - { name: foo, value: bar }
+        singleValue: ge 30
+      - meterId:
+          name: request_duration
+          tags: []
+        histogramBuckets:
+          - 5
+          - 20
+          - 50
+          - 100
+          - 500
+          - 1000
+      - meterId:
+          name: request_duration_with_label
+          tags:
+            - { name: foo, value: bar }
+        histogramBuckets:
```

Comment:
Why the histogram only have bucket, but no values?

Reply:
I saw in the error log of CI/plugin-test that I need to add a bucket of `0.0` in `expected.yaml`, which is easy to solve.

At the same time, I also saw a field: `histogramValues `, do I need to configure this? I don't seem to find this field in the documentation [ref doc](https://skywalking.apache.org/docs/skywalking-java/next/en/setup/service-agent/java-agent/plugin-test/#expecteddatayaml). Can we configure it directly?

Reply:
I think you must configure the values in the excepted file, otherwise we cannot known the test is success or not.

Reply:
Done